sdks/java/core/src/main/java/org/apache/beam/sdk/io/AvroIO.java

```diff
@@ -875,6 +926,13 @@ public void populateDisplayData(DisplayData.Builder builder) {
           recordClass,
           schemaSupplier.get());
     }
+
+    private static class JsonToSchema implements Function<String, Schema>, Serializable {
```

Comment:
AVRO-1852 to the rescue!  Just for reference: schemas are directly serializable from Avro 1.9.1+ (sharing the happiness!)

Reply:
:) yaaayy !